### DISCUSSION TITLE
Google tag manager fail my cypress test

### DISCUSSION THREAD

**Discussion by desdamo** [cypress-io/cypress#17007] [Questions and Help]
Hello everybody,

Do you already meet this issue with Google tag manager / cypress / React Js ? when i add the GTM script in the head and body (directly in html or with a npm package), lot of my cypress test failed... i tried to comment my tag and the test pass directly, i have no error mothing but the tests didnt work. do you have any idea to fix it??
Thanks

**Comment by herecydev** [comment]
Works fine for me. What does the script look like? Does it fail if you go to your application normally?
